### PR TITLE
css cleanup

### PR DESCRIPTION
**Pull request by xi**
Some more CSS  cleanup. this one is mostly about moving files.

This continues #2481 and #2502. I plan to continue creating pull requests about CSS refactoring in small steps.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/a3.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
 @import "general/layout";
 @import "general/print";
 
-@import "components/type";
+@import "components/base";
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/a3.scss]: nice.

File: src/meinberlin/meinberlin/static/stylesheets/scss/_theme.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @import "general/layout_theme";
+@import "components/base_theme";
```
Comment (der-john) [inline src/meinberlin/meinberlin/static/stylesheets/scss/_theme.scss]: coincidentally, this is ambiguous. but I like "base" nevertheless.

File: src/s1/s1/static/stylesheets/scss/components/_base_theme.scss [hunk 3]
```diff
@@ -1,4 +1,3 @@
 html {
-    @include rem-baseline;
```
Comment (der-john) [inline src/s1/s1/static/stylesheets/scss/components/_base_theme.scss]: I'm not up-to-date on s1 - but this isn't part of renaming.

**Comment by der-john** [comment]
looks good & works.

File: src/s1/s1/static/stylesheets/scss/components/_base_theme.scss (hunk 3 above)
Comment (xi) [inline src/s1/s1/static/stylesheets/scss/components/_base_theme.scss]: This property was a duplicate (also in core) so I removed it. I agree that it does not really belong into this commit.

**Pull request merged**